diff --git a/node_modules/react-spring/native.js b/node_modules/react-spring/native.js
index 736b7ec..e121b3e 100644
--- a/node_modules/react-spring/native.js
+++ b/node_modules/react-spring/native.js
@@ -9,9 +9,9 @@ class Animated {
     this.children = [];
   }
 
-  getAnimatedValue() {
+   getAnimatedValue() {
     return this.getValue();
-  }
+   }
 
   getPayload() {
     return this.payload || this;
